apps/antalmanac/src/components/Calendar/Toolbar/CustomEventDialog/DaySelector.tsx

```diff
@@ -30,24 +26,29 @@ const DaySelector = ({ days = [false, false, false, false, false, false, false],
             sx={{
                 display: 'flex',
                 flexDirection: 'row',
-                justifyContent: 'center',
-                padding: 5,
+                justifyContent: 'space-between',
+                marginTop: 10,
+                marginBottom: 10,
+                marginLeft: -5,
+                marginRight: -5,
             }}
         >
```

Comment:
```suggestion
                    style={{ gap: '10px' }}
        >
```

suggestion: the answer to the above question (where is this weird spacing on the buttons coming from?) is the margin placed upon each button, which results in spacing on the lefthand and righthand side of the button group. Using `gap` is a more maintainable, cleaner option which only places the spacing between elements